ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh

```diff
@@ -40,9 +40,52 @@ function wait_until_gpu_pods_are_running() {
 
 }
 
+function rerun_accelerator_migration() {
+#As we are adding the GPUs after installing the operator, those GPUs are not discovered automatically.
+#In order to rerun the migration we need to
+#1. Delete the migration configmap
+#2. Delete the dashboard replicaset to trigger new pods
+#Context: https://github.com/opendatahub-io/odh-dashboard/issues/1938
+
+  local timeout_seconds=600
+  local sleep_time=5
+
+  echo "Deleting configmap migration-gpu-status"
+  if ! oc delete configmap migration-gpu-status -n redhat-ods-applications;
+    then
+      printf "ERROR: When trying to delete the migration-gpu-status configmap\n"
+      return 1
+  fi
+
+  dashboard_rs=$(oc get rs -n redhat-ods-applications | grep rhods-dashboard- | awk '{print $1;exit}')
+  echo "Deleting ReplicaSet $dashboard_rs"
+  if ! oc delete rs $dashboard_rs  -n redhat-ods-applications;
+    then
+      printf "ERROR: When trying to delete the dashboard replica set\n"
+      return 1
+  fi
```

Comment:
you could probably try with `oc rollout latest dc/odh-dashboard -n redhat-ods-applications`

Reply:
Do not apply anymore